content/md/articles/cookbooks/cli_build_projects.md

```diff
@@ -252,7 +267,9 @@ we will pass `:aliases` in the `opts` and use that to construct the
 basis and also to retrieve both the `:main` class to run and the `:main-args`
 we want to use with it.
 
-We will need to use `tools.deps` to process the aliases, so that we can
+**Require `clojure.tools.deps`**
+
+We will need to use `clojure.tools.deps` to process the aliases, so that we can
```

Comment:
I had used `tools.deps` to indicate the library name (like `tools.build`) -- and the library name is used elsewhere so, while the **Require** header should use the ns, this line should still use the library name.

Reply:
I think that distinction is obvious if you're familiar with both. If that's the case, I would be even more explicit. Maybe something like:

**Require `clojure.tools.deps`**

We will need to use the`clojure.tools.deps` namespace from `tools.deps`  to process the aliases, so that we can

Reply:
That sounds good.